R/utils.R

```diff
@@ -0,0 +1,6 @@
+# yoinked from reticulate ->
+# https://github.com/rstudio/reticulate/blob/fe0eda154a80b22c0d45e043b74390b73ab8b64e/R/utils.R#L49
+yoink <- function(package, symbol) {
+    do.call(":::", list(package, symbol))
+}
+disable_conversion_scope <- yoink("reticulate", "disable_conversion_scope")
```

Comment:
That's a neat trick! I think this should be fine but if it's safer (and licensing works) I guess we could copy the whole function.

Reply:
Copying the function would also be fine, but then we need to copy the function above that as well